eng/pipelines/templates/jobs/sdk-job-matrix.yml

```diff
@@ -35,10 +35,12 @@ parameters:
     # See: https://github.com/dotnet/sdk/issues/40935
     runTests: false
   - categoryName: ContainerBased
-    container: fedora39
+    container: fedora41
     # No fedora Helix container is available, so use the ubuntu one instead.
     helixTargetContainer: $(helixTargetContainerPrefix)ubuntu-22.04-helix-amd64
     osProperties: $(linuxOsPortableProperties)
+    env:
+      OPENSSL_ENABLE_SHA1_SIGNATURES: 1
```

Comment:
@marcpopMSFT 
This parameter set concept doesn't support setting something like this. It only supports input parameters, not environment variables. You'd have to add a way to do this specifically. This is just an example, but you can see how it is done on testing in this case. I don't know what needs this environment variable, building or testing.

https://github.com/dotnet/sdk/blob/0efdc7a79cbe4ab84d0d41efe77d5985949f0e6c/eng/pipelines/templates/jobs/sdk-build.yml#L154-L159